ios/browser/api/ads/brave_ads.mm

```diff
@@ -1785,6 +1785,17 @@ - (void)toggleDislikeAd:(NSString*)creativeInstanceId
   ads->ToggleDislikeAd(brave_ads::AdHistoryItemToValue(ad_history_item));
 }
 
+- (void)clearHistory:(void (^)(BOOL success))completion {
+  if (![self isServiceRunning]) {
+    return completion(/*success=*/false);
+  }
+
+  ads->ClearHistory(base::Time(), base::Time::Max(),
+                    base::BindOnce(^(const bool success) {
+                      completion(success);
+                    }));
```

Comment:
You may be able to shorten this to `base::BindOnce(completion)` depending on if clang considers `bool` and `BOOL` different types

Reply:
Fixed